.pre-commit-config.yaml

```diff
@@ -36,31 +36,24 @@ repos:
 
   - repo: local
     hooks:
-      - id: poetry-install
-        name: sync poetry deps
-        description: Ensure development tools are installed.
-        entry: poetry install --sync
-        language: system
-        files: poetry.lock
-        pass_filenames: false
```

Comment:
Do we want to `uv sync` here?

Reply:
`sync` is done automatically each time you call `uv run`.
In CI we add the extra step to have more feedback and pass `-locked` as needed, but here it should be instantaneous.